Infrastructure-Scripts/Set-AppRoleAssignments/tools/Helpers.psm1

```diff
@@ -46,6 +45,35 @@ function New-AppRegistration {
     az ad sp create --id $IdentifierUri --output none 2>$null
 }
 
+function Set-AzureCLIAccess {
+    Param(
+        [Parameter(Mandatory = $true)]
+        [String]$IdentifierUri,
+        [Parameter(Mandatory = $true)]
+        [String]$ServicePrincipalObjectId
+    )
+
+    #Authorize Azure CLI to call app registration and acquire a token
+    #https://docs.microsoft.com/en-us/graph/api/resources/preauthorizedapplication?view=graph-rest-1.0
+    $AppRegistrationObject = (az ad app show --id $IdentifierUri | ConvertFrom-Json)
```

Comment:
May want error/null handling for this object

Reply:
Or could the ServicePrincipalObject be passed in from Get-ServicePrincipal (Line #78 Set-AppRoleAssignments.ps1)